test/_init_tests.js

```diff
@@ -134,9 +141,10 @@ class TestConfig {
     return this.skip(this.runOnChrome);
   }
 
-  skipOldChrome() {
+  skipChromeDev() {
     return this.skip(() => {
-      return this.platform.isChrome() && this.platform.getMajorVersion() < 48;
+      return this.platform.isChrome() &&
+        this.platform.getMajorVersion() > latestVersion.chrome;
```

Comment:
I assume it's guaranteed that the major version is incremented for every Chrome dev release?

Reply:
Not sure that it's guaranteed. I can update this to check against the BUILD number, which is always increasing.

Reply:
Either works. I was just curious. Can fine tune this if needed in a follow up PR.